CMakeLists.txt

```diff
@@ -2929,6 +2929,15 @@ if(ENABLE_PLUGIN_GO)
             DESTINATION usr/libexec/netdata/plugins.d)
 endif()
 
+#
+# otel.plugin
+#
+
+install(FILES
```

Comment:
@Ferroin the idea would be to build the otel.plugin prior to running CMake and then install the generated file at the proper location. I'm open to any suggestions/thoughts though.

Reply:
I'd prefer something that follows the KISS principle to an extreme. At least until we get a better idea what a proper integration would look like. For source builds, skipping the plugin if `cargo` is not available should be fine as a first step.